src/pages/NFTs/OpenBidNFT/BidModal.tsx

```diff
@@ -204,25 +276,66 @@ export const BidModal: FC<{ setVisible: (x: boolean) => void; visible: boolean }
                 <img className="m-icon" src={`${process.env.PUBLIC_URL}/img/assets/bid-success-icon.svg`} alt="" />
               </Col>
             </Row>
-            <div>Genesis #3886, Solcities</div>
-            <div>My bid: 150.5 SOL (up to 160.5 SOL)</div>
+            <div>{details?.nft_name}</div>
+            <div>My bid: {`${bidPrice} (up to ${1.03 * Number(bidPrice)})`}</div>
           </MESSAGE>
         )
       })
-    }, 1000)
+    } else {
+      notify({
+        type: 'error',
+        message: (
+          <MESSAGE>
+            <Row className="m-title" justify="space-between" align="middle">
+              <Col>Live auction bid error!</Col>
+              <Col>
+                <img className="m-icon" src={`${process.env.PUBLIC_URL}/img/assets/close-white-icon.svg`} alt="" />
+              </Col>
+            </Row>
+            <div>Please try again, if the error persists please contact support.</div>
+          </MESSAGE>
+        )
+      })
+    }
+    // // TODO: Fake API
+    // setTimeout(() => {
+    //   setIsLoading(false)
+    //   setVisible(false)
+    //   setMode('bid')
+    //   notify({
+    //     message: (
+    //       <MESSAGE>
+    //         <Row className="m-title" justify="space-between" align="middle">
+    //           <Col>Live auction bid sucessfull!</Col>
+    //           <Col>
+    //             <img className="m-icon" src={`${process.env.PUBLIC_URL}/img/assets/bid-success-icon.svg`} alt="" />
+    //           </Col>
+    //         </Row>
+    //         <div>Genesis #3886, Solcities</div>
+    //         <div>My bid: 150.5 SOL (up to 160.5 SOL)</div>
+    //       </MESSAGE>
+    //     )
+    //   })
+    // }, 1000)
   }
 
+  // TODO: change usd approx to real figure using conversion rate
+
   return (
     <PURCHASE_MODAL setVisible={setVisible} title="" visible={visible} onCancel={onCancel}>
       <div className="bm-title">You are about to purchase a</div>
-      <Row className="bm-title" align="middle" gutter={4}>
-        <Col className="bm-title-bold">(Name of the NFT)</Col>
+      <Row className="bm-title" align="middle" justify="center" gutter={4}>
+        <Col className="bm-title-bold">{details?.nft_name || '(Name of the NFT)'}</Col>
         <Col>by</Col>
-        <Col className="bm-title-bold">(Name of the artist)</Col>.
+        <Col className="bm-title-bold">(Name of the artist).</Col>
```

Comment:
I couldn't figure where to get the name of artist, which I'm guessing is owner's name the nearest I could find is the mint_address

Reply:
For future reference, its `nftMetadata.properties.creators` from the `useNFTDetails` context